l1-contracts/src/storage/UniFiAVSManagerStorage.sol

```diff
@@ -22,6 +22,12 @@ abstract contract UniFiAVSManagerStorage {
         mapping(uint256 => uint8) chainIdToBitmapIndex;
         // Set of allowlisted restaking strategies
         EnumerableSet.AddressSet allowlistedRestakingStrategies;
+        // Mapping to store validator registration data
+        mapping(bytes32 => ValidatorRegistrationData) validatorRegistrations;
```

Comment:
can you use named mappings everywhere like this `mapping(address operator => InvalidValidator[] invalidValidators) slashedOperators;`?